deployment/network-operator/charts/sriov-network-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: sriov-network-operator
-version: 0.1.0
-kubeVersion: '>= 1.16.0-0'
-appVersion: 1.2.0
+version: 0.0.0-dev
```

Comment:
Do we want this change?

Reply:
let's revert it for now and discuss how do we want to implement chart sync